### DISCUSSION TITLE
[RELEASE] Pump

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#166] [Map Releases]
## This is a manual migration of forum user Naievil & oldschool125's map release `Pump`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t2017-release-pump).

**This project was formerly known as "Station."**

Hey guys,

[Anass-nzp125](https://nzportable.forumotion.com/u10007) and I have finished a map that is based off of a run-down city. This map features a gas station and several surrounding buildings. The gas station lights were supposed to be brighter once the power is on, but that's just not going to happen.

This map features several of the basic perks, but still lacks a Pack-a-Punch and a Mystery Box. The city is covered in an immense fog, so be prepared for that. Previous revisions of this map did not include this.

You will need the jeep.mdl, like what Land v1.2 featured; I have added it in the download link.
This file is, once again, created by the dev team and neither one of us takes credit for it.

This file also includes a customized loading screen, which looks pretty choppy, but will do for now.

Screenshots:
![shot_0](https://user-images.githubusercontent.com/26030250/216206387-b13e940d-63ce-4b76-8e4e-6061ec45c2c2.jpg)
![shot_1](https://user-images.githubusercontent.com/26030250/216206385-fc982f73-c70f-4a6b-a088-703e746e9dc5.jpg)
![shot_2](https://user-images.githubusercontent.com/26030250/216206384-f972243d-32d6-40e7-8f63-5da5535e9a52.jpg)
![shot_3](https://user-images.githubusercontent.com/26030250/216206380-e6c0251f-b23f-49f2-8e85-e32fd443de40.jpg)
![shot_4](https://user-images.githubusercontent.com/26030250/216206377-cc12d1c0-1af8-4647-a073-f257895cf163.jpg)
![shot_5](https://user-images.githubusercontent.com/26030250/216206374-bb5691d7-4a74-43d6-84ea-db8e3a251173.jpg)

This map is designed to work on PSP using the 1.02 Unofficial Patch. Get it [here](https://nzportable.forumotion.com/t1976-patch-nzp-unofficial-patch-demo-1-0-2). ***Please note that this map may crash occasionally on PSP. PC is good to go***

To fully enjoy this new release and to not clog your map folder, please delete the gas_station.way and gas_station.bsp

Thanks, enjoy the release!

If you would like to play the map so far, please use these download links:

# [DOWNLOAD](https://archive.org/download/nzp_pump/nzp_pump.zip)

**Comment by Naievil** [comment]
Haven't seen the Pump screenshots in probably 7 years. Thanks for the migration.

**Comment by Phynix0027** [comment]
Is the only area on the map the gas station and the building with the switch, or are there more doors?

**Comment by oshogaps-glitch** [comment]
idk why if im on pc or not but this map glitches zombies get stuck and float in air....

when i go upstairs it makes it very hard for zombies to find me ...

the spawn room door in front zombies get stuck in their like really bad 

if you would like a video i can post
